inginious/frontend/pages/course_admin/submissions.py

```diff
@@ -154,9 +155,15 @@ def get_submissions(self, course, user_input):
         except ValueError:  # If match of datetime.strptime() fails
             pass
             
-        # Mongo operations
-        data = list(self.database.submissions.find({**query_base, **query_advanced}).sort([(user_input.sort_by, 
-            pymongo.DESCENDING if user_input.order == "0" else pymongo.ASCENDING)]))
+        # Using submission manager to retrieve data
+        fields = ["username","best","result","submitted_on","grade","status","taskid"]
+        fields_dict = {field : 1 for field in fields}
+        if is_for_table:
+            data = list(self.database.submissions.find({**query_base, **query_advanced},fields_dict).sort([(user_input.sort_by, 
+                         pymongo.DESCENDING if user_input.order == "0" else pymongo.ASCENDING)]))
+        else: 
+            data = list(self.database.submissions.find({**query_base, **query_advanced}).sort([(user_input.sort_by, 
+                         pymongo.DESCENDING if user_input.order == "0" else pymongo.ASCENDING)]))
```

Comment:
Which use cases require to retrieve the complete submission list including all data in the collection (i.e. when `is_for_table=False`)?